### ISSUE TITLE
I am getting "Floating point exception (core dumped)" when training the model.  How can I fix this?

### ISSUE THREAD

**Issue by ylevental** [SKrisanski/FSCT#8]
I don't even know where this is happening since it doesn't tell me where.  I am using all the example.las files that you provided.

**Comment by SKrisanski** [comment]
I'll need a lot more information to help you here. Full python console output at a minimum. Also, if you are training on example.las,this is not sufficient for a useful model. These files do work to train a test model and to verify correct usage/installation, but you need a lot more labeled data than that for a real model. They are there just for testing and as an example of how the training data needs to be formatted. If you put something with a different file structure in, it will probably fail in some way.

**Comment by ylevental** [comment]
@SKrisanski Great news!  I found that I had to change to device='cuda' from device='cpu' in the parameters!

Also, I thought I had to uncomment `# self.device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')` but I had to change the running_point_cloud_vis line to the following:

`running_point_cloud_vis = np.vstack((running_point_cloud_vis, np.hstack((data.pos.cpu() + np.array([i * 7, 0, 0]), data.y.cpu().T, preds.cpu().T))))`

There are some other errors, but this is a major step forward!
